tests/pytest/test_admin.py

```diff
@@ -0,0 +1,58 @@
+import pytest
+from django.conf import settings
+from django.contrib.auth.models import Group, User
+from benefits.admin import BenefitsAdminSite
+
+
+@pytest.fixture
+def model_User():
+    return User.objects.create(is_active="1", is_staff="1")
```

Comment:
Not at all, this makes sense to use `True` 👍 I was looking at the database and got confused because I saw `True` gets stored as `1`, but we are working with the Django model, not the database value and we should use `True` for setting the boolean.